openstack_dashboard/test/selenium/integration/test_user_credentials.py

```
@@ -0,0 +1,175 @@
+        'credentials',
+    ))
+    driver.get(url)
+    filter_input_field = driver.find_element_by_css_selector(".form-control")
+    filter_input_field.clear()
+    filter_input_field.send_keys('demo')
+    driver.find_element_by_id("credentialstable__action_filter").click()
+    try:
+        driver.find_element_by_xpath(
+            "//td[text()='No items to display.']")
+    except(exceptions.NoSuchElementException):
+        assert False, "Message 'No items to display' not found"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)